.github/workflows/test-suite.yml

```diff
@@ -23,6 +23,37 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  lint:
```

Comment:
I'm wondering if making it a separate workflow wouldn't be better 🤔 Then we could limit when this job is triggered to only `apps/test-suite` files. What do you think?
I also don't like workflows with multiple jobs, because if any of them fails and you want to restart it, you must restart all of them 🤷‍♂️

Reply:
Yeah, I agree with this suggestion. I was also considering creating separate workflow to reduce the `runs-on` paths, but at the end I have decided to push it as additional step in the existing workflow. I will extract lint to the separate workflow soon. 🙂